client/src/components/Layout/UserHome/UserHome.css

```diff
@@ -0,0 +1,131 @@
+.component_continer{
```

Comment:
Change the CSS file to camelCase name style

Reply:
done